lib/block-supports/layout.php

```diff
@@ -54,23 +56,29 @@ function gutenberg_get_layout_style( $selector, $layout, $has_block_gap_support
 		$wide_max_width_value = wp_strip_all_tags( explode( ';', $wide_max_width_value )[0] );
 
 		if ( $content_size || $wide_size ) {
-			$style  = "$selector > :where(:not(.alignleft):not(.alignright)) {";
-			$style .= 'max-width: ' . esc_html( $all_max_width_value ) . ';';
-			$style .= 'margin-left: auto !important;';
-			$style .= 'margin-right: auto !important;';
-			$style .= '}';
-
-			$style .= "$selector > .alignwide { max-width: " . esc_html( $wide_max_width_value ) . ';}';
-			$style .= "$selector .alignfull { max-width: none; }";
+			$declarations[ "$selector > :where(:not(.alignleft):not(.alignright))" ] = array(
+				'max-width'    => $all_max_width_value,
+				'margin-left'  => 'auto !important;',
+				'margin-right' => 'auto !important;',
```

Comment:
It looks like the extra semi-colons get stripped anyway, but for neatness sake should we remove them here?

```suggestion
				'margin-left'  => 'auto !important',
				'margin-right' => 'auto !important',
```

Reply:
Yes! I missed those 2, thank you for catching that!